virttest/vt_utils/block_disk.py

```
@@ -0,0 +1,408 @@
+    return False
+
+
+def mount(src, dst, fstype=None, options=None, verbose=False):
+    """
+    Mount src under dst. Even if it has been mounted already,
+    remout will be done with options.
+
+    :param src: source device or directory
+    :type src: string
+    :param dst: mountpoint
+    :type dst: string
+    :param fstype: filesystem type
+    :type fstype: string
+    :param options: mount options, which should be separated by ","
+                    between each options
+    :type options: string
+    :param verbose: output the verbose if it's True
+    :type verbose: boolean
+
+    :return: if mounted successfully, return True. Otherwise, return False
+    :rtype: boolean
+    """
+    options = '' if options is None else options
+    if is_mount(src, dst, fstype, options, verbose):
+        if 'remount' not in options:
+            options = 'remount,%s' % options
+    cmd = ['mount']
+    if fstype:
+        cmd.extend(['-t', fstype])
+    if options:
+        cmd.extend(['-o', options])
+    cmd.extend([src, dst])
+    cmd = ' '.join(cmd)
+    return process.system(cmd, verbose=verbose) == 0
+
+
+def umount(src, dst, fstype=None, verbose=False):
+    """
+    Umount src from dst, if src really mounted under dst.
+
+    :param src: source device or directory
+    :type src: string
+    :param dst: mountpoint
+    :type dst: string
+    :param fstype: filesystem type
+    :type fstype: string
+    :param verbose: output the verbose if it's True
+    :type verbose: boolean
+
+    :return: if umount successfully, return True. Otherwise, return False
+    :rtype: boolean
+    """
+    mounted = is_mount(src, dst, fstype, verbose=verbose)
+    if mounted:
+        from virttest import utils_package
+        package = "psmisc"
+        # check package is available, if not try installing it
+        if not utils_package.package_install(package):
+            LOG.error("%s is not available/installed for fuser", package)
```

Looks we need to raise an exception instead, maybe?